### PR TITLE
Add isFocused as props on placeholder component

### PR DESCRIPTION
**Pull request by Nelrohd**

### CODE REVIEW

File: src/Select.js [hunk 1]
```diff
@@ -1396,7 +1396,12 @@ export default class Select extends Component<Props, State> {
 
     if (!this.hasValue() || !controlShouldRenderValue) {
       return inputValue ? null : (
-        <Placeholder {...commonProps} key="placeholder" isDisabled={isDisabled}>
+        <Placeholder 
+          {...commonProps} 
+          key="placeholder"
+          isDisabled={isDisabled}
+          isFocused={focusedValue !== null}
```
Comment (mike1808) [inline src/Select.js]: there is `isFocused` var in state and you probably should use it

**Comment by Nelrohd** [comment]
Thanks @mike1808

**Comment by emmya** [comment]
🙏

**Comment by gwyneplaine** [comment]
this looks good, thanks @Nelrohd

**Pull request merged**